claimer/src/op/index.ts

```diff
@@ -1,8 +1,5 @@
-import {
-  buildProveWithdrawal,
-  getWithdrawalHash,
-} from '@relay-protocol/helpers'
-import { Portal2 } from '@relay-protocol/helpers/abis'
+import { buildProveWithdrawal } from '@relay-protocol/helper-bedrock'
```

Comment:
I think we want to mention Op in the name of the function itself as we may have to import it for many networks? Or we use the same name for all claimer helpers and we use named imports/aliases ?

Reply:
> I think we want to mention Op in the name of the function itself as we may have to import it for many networks? Or we use the same name for all claimer helpers and we use named imports/aliases ?

Well no, the point would be that we can then call the same function no matter the network. This creates an abstraction.